bindings/nodejs/lib/types/models/block-metadata.ts

```diff
@@ -13,35 +15,19 @@ export interface IBlockMetadata {
      */
     blockId: HexEncodedString;
     /**
-     * The parent block ids.
-     */
-    parents: HexEncodedString[];
-    /**
-     * Is the block solid.
-     */
-    isSolid: boolean;
-    /**
-     * Is the block referenced by a milestone.
+     * The block state.
      */
-    referencedByMilestoneIndex?: number;
+    blockState?: BlockState;
```

Comment:
```suggestion
    blockState: BlockState;
```
Not optional. Can you do the same in Python and remove the TODO please ?

Reply:
sure